app/src/ui/thank-you/thank-you.tsx

```diff
@@ -80,14 +80,11 @@ export class ThankYou extends React.Component<IThankYouProps, {}> {
               renderUrlsAsLinks={true}
             />
           </div>
-          <div className="thank-you-note">
-            The Desktop team wants to thank you personally.
-          </div>
         </div>
       </div>
     )
 
-    const thankYou = 'Thank you for all your hard work on GitHub Desktop'
+    const thankYou = "Thanks so much for all your hard work on GitHub Desktop. We're so grateful for you helping to make the app better for everyone!"
```

Comment:
This is the TODO mentioned in the PR body, where adding the second sentence messed up the version number. I think basically we just need to inject something like the result of the if statement below this into the end of the first sentence before the period.